[TA] Updates for 2022-04-01 documentations (#28923)
Azure/azure-sdk-for-java: sdk/textanalytics/azure-ai-textanalytics/README.md

> # Azure Language service client library for Java

Reviewer: I think this should stay "Text Analytics" since that's still the name of the client library, it just uses the language service

> and includes the main features below:

Reviewer 2: Consider updating the description of the service here too. Something like https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/README.md

- Sentiment Analysis
- Entity Recognition (Named, Linked, and Personally Identifiable Information (PII) entities)
- Language Detection
- Key Phrase Extraction
- Multiple Actions Analysis Per Document
- Healthcare Entities Analysis
- Extractive Text Summarization
- Custom Named Entity Recognition
- Custom Text Classification

> [Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_reference_doc] | [Product Documentation][product_documentation] | [Samples][samples_readme]

Reviewer 2: Product Documentation link needs to be updated

> 

Reviewer 2: the prerequisites section needs to be updated too

If you want to take dependency on a particular version of the library that is not present in the BOM,
add the direct dependency to your project as follows.

[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
```xml
<dependency>
    <groupId>com.azure</groupId>
    <artifactId>azure-ai-textanalytics</artifactId>

>     <version>5.2.0-beta.3</version>

Reviewer 2: curios, this goes form a GA to a beta. does Java usually show the latest? or the latest stable library

Author: Usually java shows the latest stable version in the main and release the beta in feature branch. The feature branch will merge to main branch at GA. But since TA is merging the latest beta changes to main branch. I will update the version to beta for the beta features, otherwise the custom features should not include in the README.

</dependency>
```
[//]: # ({x-version-update-end})
**Note:** This version of the client library defaults to the `2022-04-01-preview` version of the service.

This table shows the relationship between SDK services and supported API versions of the service:

|SDK version|Supported API version of service
|-|-

> |5.2.x | 3.0, 3.1, 2022-04-01-preview (default)

Reviewer 2: Consider adding a note to your changelog about dropping support for `3.2-preview.2`

Reviewer 2: Ahh I was looking at main. thank you

|5.1.x | 3.0, 3.1 
|5.0.x | 3.0

#### Create a Cognitive Services or Language Service resource
The Language service supports both [multi-service and single-service access][service_access]. Create a Cognitive Services 
resource if you plan to access multiple cognitive services under a single endpoint/key. For Language service access only,
create a Language service resource.

You can create either resource using the 

**Option 1:** [Azure Portal][create_new_resource] 

**Option 2:** [Azure CLI][azure_cli]

Below is an example of how you can create a Language service resource using the CLI:

```bash
# Create a new resource group to hold the Language service resource -
# if using an existing resource group, skip this step
az group create --name <your-resource-group> --location <location>

> ```

Reviewer 2: comment in the bash section needs to be updated

For more information about creating the resource or how to get the location and sku information see [here][azure_cli]

### Authenticate the client

> In order to interact with the Language service, you will need to create an instance of the Language service client,

Reviewer: ```suggestion
In order to interact with the Language service, you will need to create an instance of the Text Analytics client,
```

both the asynchronous and synchronous clients can be created by using `TextAnalyticsClientBuilder` invoking `buildClient()`
creates a synchronous client while `buildAsyncClient()` creates its asynchronous counterpart.

You will need an **endpoint** and either a **key** or **AAD TokenCredential** to instantiate a client object. 

#### Looking up the endpoint
You can find the **endpoint** for your Language service resource in the [Azure Portal][azure_portal] under the
"Keys and Endpoint", or [Azure CLI][azure_cli_endpoint].
```bash

> # Get the endpoint for the text analytics resource

Reviewer 2: ```suggestion
# Get the endpoint for the Language service resource
```

az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "endpoint"
```

> #### Create a Language service client with key credential

Reviewer: revert here as well (edit: there's a few instances on the README where we want to keep TA when referring to the client.

For samples on using the production recommended option `ExtractKeyPhrasesBatch` see [here][extract_key_phrases_sample].
Please refer to the service documentation for a conceptual discussion of [key phrase extraction][key_phrase_extraction].

### Recognize named entities
Run a predictive model to identify a collection of named entities in the provided document or batch of documents and 
categorize those entities into categories such as person, location, or organization.  For more information on available

> categories, see [Text Analytics Named Entity Categories][named_entities_categories].

Reviewer 2: ```suggestion
categories, see [Named Entity Categories][named_entities_categories].
```

});
```
For samples on using the production recommended option `RecognizeLinkedEntitiesBatch` see [here][recognize_linked_entities_sample].
Please refer to the service documentation for a conceptual discussion of [entity linking][entity_linking].

### Analyze healthcare entities
Language service for health is a containerized service that extracts and labels relevant medical information from 
unstructured texts such as doctor's notes, discharge summaries, clinical documents, and electronic health records.

> For more information see [How to: Use Language service for health][healthcare].

Reviewer: I'm pretty sure the service still refers to this as TA for Health
